geoips/sector_utils/tc_tracks.py

```diff
@@ -192,8 +192,13 @@ def set_tc_area_def(
         template_func_name = template_dict["area_def_generator_func"]
         template_args = template_dict["area_def_generator_args"]
     except:
-        template_func_name = template_dict["spec"]["sector_spec_generator"]["name"]
-        template_args = template_dict["spec"]["sector_spec_generator"]["arguments"]
+        # I think this is probably what we will want.
+        # template_func_name = template_dict["spec"]["sector_spec_generator"]["name"]
+        # template_args = template_dict["spec"]["sector_spec_generator"]["arguments"]
+        # This is probably not the formatting we want for the dynamic templates,
+        # but leave it for now until we finalize.
+        template_func_name = template_dict["spec"]["generators"]["spec"]["name"]
```

Comment:
@jsolbrig Again, updated this temporarily to match the dynamic template layout to make tests pass - though I think we will want to update to something more like the commented out portion.

Reply:
Yeah, we need to work on the dynamic sectors soon but if this works for now, I'm okay with it.